enterprise/server/cmd/ci_runner/main.go

```diff
@@ -499,27 +502,34 @@ func (ar *actionRunner) Run(ctx context.Context, startTime time.Time) error {
 	}
 
 	wfc := &bespb.WorkflowConfigured{
-		WorkflowId:          *workflowID,
-		ActionName:          ar.action.Name,
-		ActionTriggerBranch: *triggerBranch,
-		ActionTriggerEvent:  *triggerEvent,
+		WorkflowId:         *workflowID,
+		ActionName:         ar.action.Name,
+		ActionTriggerEvent: *triggerEvent,
+		PushedRepoUrl:      *pushedRepoURL,
+		PushedBranch:       *pushedBranch,
+		CommitSha:          *commitSHA,
+		TargetRepoUrl:      *targetRepoURL,
+		TargetBranch:       *targetBranch,
 	}
 	configuredEvent := &bespb.BuildEvent{
 		Id:      &bespb.BuildEventId{Id: &bespb.BuildEventId_WorkflowConfigured{WorkflowConfigured: &bespb.BuildEventId_WorkflowConfiguredId{}}},
 		Payload: &bespb.BuildEvent_WorkflowConfigured{WorkflowConfigured: wfc},
 	}
-	for _, bazelCmd := range ar.action.BazelCommands {
-		iid := newUUID()
-		wfc.Invocation = append(wfc.Invocation, &bespb.WorkflowConfigured_InvocationMetadata{
-			InvocationId: iid,
-			BazelCommand: bazelCmd,
-		})
-		eventID := &bespb.BuildEventId{
-			Id: &bespb.BuildEventId_WorkflowCommandCompleted{WorkflowCommandCompleted: &bespb.BuildEventId_WorkflowCommandCompletedId{
+	// If there was a merge conflict, we won't be invoking any bazel commands.
+	if !ws.mergeConflict {
+		for _, bazelCmd := range ar.action.BazelCommands {
+			iid := newUUID()
+			wfc.Invocation = append(wfc.Invocation, &bespb.WorkflowConfigured_InvocationMetadata{
 				InvocationId: iid,
-			}},
+				BazelCommand: bazelCmd,
+			})
+			eventID := &bespb.BuildEventId{
+				Id: &bespb.BuildEventId_WorkflowCommandCompleted{WorkflowCommandCompleted: &bespb.BuildEventId_WorkflowCommandCompletedId{
+					InvocationId: iid,
+				}},
+			}
+			configuredEvent.Children = append(configuredEvent.Children, eventID)
 		}
-		configuredEvent.Children = append(configuredEvent.Children, eventID)
```

Comment:
this is a bit confusing: the comment says "if merge conflict", the code says "if not merge conflict", and for someone not familiar with this code, it's tough to understand the relation between the bazel commands the comment references and what happens in the loop (sending invocation events?)

maybe you could say something like:

// If this commit merges cleanly, the runner will execute several bazel commands below to run tests. If the commit does not merge cleanly, the runner will exit early without running those commands and does not need to create invocation streams for them.

or something similar (you know what's going on here way better than I)

Reply:
Done